spec/requests/search_spec.rb

```diff
@@ -55,4 +55,32 @@
       expect(response.parsed_body).to eq("result" => mock_documents)
     end
   end
+
+  describe "GET /search/users" do
```

Comment:
Added specs for the user endpoint which I forgot to do before. Yes, shame on me.